jenkins/automation/github-release.groovy

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env groovy
+pipeline {
+
+    agent {
+        node {
+            label 'docker-centos-7.9.2009-node'
+        }
+    }
+
+    parameters {
+        string(name: 'GIT_TAG', defaultValue: '', description: 'Image tag/Commit tag required for GitHub Release')
+        string(name: 'SERVICES_VERSION', defaultValue: '', description: 'Services(cortx-k8s) version on which image deployment is tested')
+        string(name: 'CHANGESET_URL', defaultValue: '', description: 'CHNAGESET.md file url.')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for GitHub Release script', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for GitHub release script', trim: true)
```

Comment:
Can we take the destination repo for GitHub Release i.e. https://github.com/Seagate/cortx as an parameter.

Reply:
Added parameter

Reply:
Should we add this parameter also in `jenkins/automation/release-process.groovy`

Reply:
Done